qfieldcloud_sdk/sdk.py

```diff
@@ -418,6 +441,19 @@ def upload_files(
 
         Returns:
             A list of dictionaries with information about the uploaded files.
+
+        Example:
+            ```python
+            client.upload_files(
+                project_id="123e4567-e89b-12d3-a456-426614174000",
```

Comment:
```suggestion
                project_id="123e4567-e89b-12d3-a456-426614174000",
```
Hopefully this does not match an actual project in the DB :dagger:

Reply:
Hope so, too!